components/ui/Link.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types'
+import NextLink from 'next/link'
+import styled, { css } from 'styled-components'
+
+import { linkStyles } from './Typography'
+import theme from '../../util/theme'
+
+const sharedLinkStyles = css`
+  ${linkStyles}
+  justify-self: left;
+  margin: 0 auto 0 0;
+  cursor: ${props => props.theme.cursors.select};
```

Comment:
why call this style `select` and not `pointer`?

Reply:
This is a good point. :)